src/frontend/src/lib/utils/exchange.utils.ts

```diff
@@ -12,10 +12,12 @@ export const usdValue = ({
 	balance: BigNumber | undefined;
 	exchangeRate: number;
 }): number =>
-	Number(
-		formatToken({
-			value: balance ?? ZERO,
-			unitName: decimals,
-			displayDecimals: decimals
-		})
-	) * exchangeRate;
+	nonNullish(balance)
+		? Number(
+				formatToken({
+					value: balance,
+					unitName: decimals,
+					displayDecimals: decimals
+				})
+			) * exchangeRate
+		: 0;
```

Comment:
not sure if it makes sense to use ZERO constant here?

Reply:
you mean inside `Number(ZERO)` ? For sure, if you prefer. I did like that so that we have less computational work, simply `0`, eheh

Reply:
Yeah something like `ZERO.toNumber()`. I was just thinking that maybe it would be convenient to search everywhere in the app where we use zero for a feature? Not sure it's worth it, just a though. That's why the PR is already approved ;)

Reply:
ah! makes sense indeed! it could be useful at some point.